tests/dcerpc/dcerpc-3109/test.yaml

```diff
@@ -0,0 +1,16 @@
+
+# disables checksum verification
+args:
+  - -k none --set stream.midstream=true
+
+checks:
+
+  - filter:
+      count: 1
+      match:
+        event_type: dcerpc
```

Comment:
It's smb w dcerpc so the event would never be dcerpc, dcerpc will be nested inside smb event

Reply:
Will you do the changes ? Or should I ? (and the suricata fixes if any)

Reply:
I will. Thank you for this :)
I'm chasing the bug as it still persists

Reply:
What should I do with this PR ? Is there a redmine ticket reference ?

Reply:
I think it's I who needs to do things.. Will check the tickets.. Got a report on similar lines from Brandon and possibly Peter. Will review them and give the update/create the tickets. 
Sorry. this fell off the radar completely!